omnivoreql/omnivoreql.py

```diff
@@ -20,6 +20,26 @@ def __init__(self, api_token: str, graphqlEndpointUrl: str = "https://api-prod.o
         self.client = Client(transport=transport,
                              fetch_schema_from_transport=False)
 
+    def save_page(self, url: str, original_content: str):
```

Comment:
Just a question here: url seems weird! Is it optional? should we make it optional? Can we pass invalid URL or any value with our custom html content?

Reply:
url is not optional - https://github.com/omnivore-app/omnivore/blob/ae8790b5456dc0abf388e79719644e9755207724/packages/api/src/schema.ts#L577 . Reason is that you're essentially passing the source of the page yourself for a url via this api, rather than asking Omnivore to fetch the source themselves (which is what saveUrl api does). Both need url input for the same reason.

Reply:
Approved. You can do whatever you want.

![image](https://github.com/yazdipour/OmnivoreQL/assets/8194807/ea700eb7-abac-4bd6-b2f0-55f1b19d2af2)

Reply:
Yep. Can you make the package release please, so i can use it as a dependency from your repo itself? @yazdipour

Reply:
If I'm not mistaken it should already be there. Test version 0.2.2.
Please let me know if it didn't work.